packages/cozy-harvest-lib/src/services/jobUtils.js

```diff
@@ -30,3 +30,8 @@ export const waitForRealtimeEvent = (
       reject(new Error('Timeout for waitForRealtimeResult'))
     }, timeout)
   })
+
+export const sendRealtimeNotification = ({ client, data }) => {
+  const rt = new CozyRealtime({ client })
+  rt.sendNotification(JOBS_DOCTYPE, 'test', data)
+}
```

Comment:
Now that you know you can use client.plugins.realtime can you use it for the new code please?

Reply:
@Crash-- Done but the cozy-client realtime plugin does not expose the sendNotification method. Fixed with https://github.com/cozy/cozy-libs/pull/1846

Will need to upgrade cozy-realtime in home and banks after that and in peerDep of harvest after next cozy-realtime release.